doc/distrib/NodeHelpFiles/Revit.Schedules.ScheduleFilter.FilterType.md

```diff
@@ -0,0 +1,24 @@
+## In Depth
+`ScheduleFilter.FilterType` returns the method used for the input filter.
+Possible filter types include:
+
+- Equal - The field value is equal to the specified value.
+- NotEqual - The field value is not equal to the specified value.
+- GreaterThan - The field value is greater than the specified value.
+- GreaterThanOrEqual - The field value is greater than or equal to the specified value.
+- LessThan - Less The field value is less than to the specified value.
```

Comment:
Extra "Less" before "The field value"?
Also remove "to" from "less than to the specified value"

Reply:
Oops this was copied directly from the API docs, so I might let them know too. lol